arc-lang/etc/Cargo.toml.template

```diff
@@ -0,0 +1,17 @@
+[package]
+name        = "${CRATE_NAME}"
+version     = "0.0.0"
+authors     = ["The arc-mlir testing gnome"]
+edition     = "2021"
+license     = "MIT"
+description = "Generic MLIR-to-Rust test case"
+
+[dependencies]
+arc-runtime         = { version = "=0.0.0", path = "${ARC_RUNTIME_SOURCE_DIR}", features = ["legacy"] }
+hexf                = { version = "0.2.1" }
+ndarray             = { version = "0.13.0" }
+prost               = { version = "0.7.0" }
```

Comment:
I think we should remove these dependencies. I can do it in another PR.

Reply:
Don't we need arc-runtime?

Reply:
Ah, I thought Github would mark the lines. We should only need arc-runtime. It can re-export the other dependencies. I tried it but got a problem with `serde`, so maybe we will still need serde in the dependency list. I can try to fix it.

Reply:
Ok, I'll merge this then.